docs/getting-started/handbooks/console/endpoints.mdx

```diff
@@ -0,0 +1,121 @@
+---
+id: endpoints
+title: Endpoints
+sidebar_label: Endpoints
+---
+
+In this guide, we will see how to configure the various types of endpoints available in my platform console, including:
+- [CRUD](/getting-started/handbooks/console/endpoints.mdx#create-a-crud-endpoint)
+- [MongoDB View](/getting-started/handbooks/console/endpoints.mdx#create-a-mongodb-view-endpoint)
+- [Microservice](/getting-started/handbooks/console/endpoints.mdx#create-a-microservice-endpoint)
+- [External Proxy](/getting-started/handbooks/console/endpoints.mdx#create-an-external-proxy-endpoint)
+- [Cross Project Proxy](/getting-started/handbooks/console/endpoints.mdx#create-a-cross-project-proxy-endpoint)
+- [Fast Data Projection](/getting-started/handbooks/console/endpoints.mdx#create-a-fast-data-projection-endpoint)
+- [Fast Data Single View](/getting-started/handbooks/console/endpoints.mdx#create-a-fast-data-single-view-endpoint)
+
+To good deeper visit the [Endpoint](/development_suite/api-console/api-design/endpoints.md) dedicated documentation page.
+
+## Prerequisites
+
+Before you begin creating endpoints, it is necessary to have an API Gateway service installed. 
+To install it, simply create a new microservice from the marketplace and select "API Gateway" from the available service catalog.
+
+![Install API Gateway](img/install-api-gateway.png)
+
+All endpoint types have some basic fields, which are:
+- **Base Path**: is the prefix of the route. 
+    It can be set as the base address to which the API is served, relative to the host that supplies the endpoint.
+- **Description**: an optional description for the endpoint.
+
+:::caution
+Please be careful when choosing the **Base Path**, as it cannot be modified after creating the endpoint.
+:::
+
+## Create a CRUD Endpoint
+
+Creating an endpoint of type **CRUD** hooks your endpoint directly to one of your CRUDs collections defined in `Data Models/MongoDB CRUD` section in Mia-Platform Console.
+
+:::caution 
+To create a CRUD Collection it is necessary to have a CRUD Service installed, to do so you simply create a new Microservice from Marketplace choosing **CRUD Service** from the catalog.
+:::
+
+To create a CRUD endpoint you have to move in the `Endpoint` section of the Mia-Platform Console, here you can create new endpoints.
```

Comment:
```suggestion
To create a CRUD endpoint you have to move in the `Endpoint` section of the Mia-Platform Console, here you can create new endpoints.
```

This sentence is always repeated, let's put it at the beginning with a screenshot of the page (with navigation).